azext_iot/central/_help.py

```diff
@@ -512,6 +562,201 @@ def _load_central_roles_help():
     """
 
 
+def _load_central_organizations_help():
+    helps[
+        "iot central organization"
+    ] = """
+        type: group
+        short-summary: Manage and configure organizations for an IoT Central application.
+    """
+
+    helps[
+        "iot central organization list"
+    ] = """
+        type: command
+        short-summary: Get the list of organizations for an IoT Central application.
+
+        examples:
+        - name: List organizations in an application
+          text: >
+            az iot central organization list
+            --app-id {appid}
+    """
+
+    helps[
+        "iot central organization show"
+    ] = """
+    type: command
+    short-summary: Get the details of a organization by ID.
+    examples:
+      - name: Get details of organization
+        text: >
+          az iot central organization show
+          --app-id {appid}
+          --org-id {organizationId}
+    """
+
+    helps[
+        "iot central organization delete"
+    ] = """
+    type: command
+    short-summary: Delete an organization by ID.
+    examples:
+      - name: Delete an organization
+        text: >
+          az iot central organization delete
+          --app-id {appid}
+          --org-id {organizationId}
+    """
+
+    helps[
+        "iot central organization create"
+    ] = """
+    type: command
+    short-summary: Create an organization in the application.
+    examples:
+      - name: Create an organization
+        text: >
+          az iot central organization create
+          --app-id {appid}
+          --org-id {organizationId}
+
+      - name: Create an organization, child of a parent one in the application.
+        text: >
+          az iot central organization create
+          --app-id {appid}
+          --org-id {organizationId}
+          --parent-id {parentId}
+        """
+
+
+def _load_central_jobs_help():
+    helps[
+        "iot central job"
+    ] = """
+        type: group
+        short-summary: Manage and configure jobs for an IoT Central application.
+    """
+
+    helps[
+        "iot central job list"
+    ] = """
+        type: command
+        short-summary: Get the list of jobs for an IoT Central application.
+
+        examples:
+        - name: List jobs in an application
+          text: >
+            az iot central job list
+            --app-id {appid}
+    """
+
+    helps[
+        "iot central job create"
+    ] = """
+    type: command
+    short-summary: Create and execute a job via its job definition.
+    examples:
+      - name: Create a job with name
+        text: >
+          az iot central job create
+          --app-id {appid}
+          --job-id {jobId}
```

Comment:
I find it strange to have both `--job-id` and `--job-name` for the same command. Can `--job-name` be changed to `--display-name` or similar if that is what it is intended for.

Reply:
this is consistent to other commands (e.g. --device-id, --device-name). we'd like to reduce breaking changes at minimum and eventually align all in a different release